src/harness/util.py

```diff
@@ -323,26 +321,33 @@ def computePropagationAntennaModel(request):
         response = 400
         return response
     
-# ITM pathloss (if receiver type is FSS) or the hybrid model pathloss (if receiver type is PPA) and corresponding antenna gains.
-# The test specification notes that the SAS UUT shall use default values for w1 and w2 in the ITM model.
+    # ITM pathloss (if receiver type is FSS) or the hybrid model pathloss (if receiver type is PPA) and corresponding antenna gains.
+    # The test specification notes that the SAS UUT shall use default values for w1 and w2 in the ITM model.
     result = {}
     
     if isfss:
-        path_loss = wf_itm.CalcItmPropagationLoss(tx['latitude'], tx['longitude'], tx['height'], rx['latitude'], rx['longitude'], rx['height'], reliability=reliability_level, freq_mhz=3625.)
+        path_loss = wf_itm.CalcItmPropagationLoss(tx['latitude'], tx['longitude'], tx['height'], \
```

Comment:
No need of the backslash, here and below. expression within parenthesis can be put on several lines.

Reply:
True. It's just our preference

Reply:
When I give "cosmetic recommendation", it is only when the code breaks the code style guidelines of Winnforum which are based on Python PEP-8 with some small addition.
Up to you at the end.
According to PEP-8:
> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

See :  https://www.python.org/dev/peps/pep-0008/